client/app/components/onboardingGuide/GuideFilterPage.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideFilterPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
```

Comment:
I really want to make every component independent what comes to styles. So, when we relocate components, they don't need to worry about styles (of parent-component).
It's also annoying when old code defines styles in a form of 
```
.component {
  p {
    color: black;
  }
}
```

That means that all the injected Childs are going to have that  generic `<p>` style as well.
=> I prefer adding className every time there's styles attached to that component.